packages/services/api/src/modules/integrations/resolvers/Organization.ts

```diff
@@ -35,10 +35,9 @@ export const Organization: Pick<
     });
   },
   viewerCanModifyGitHubIntegration: async (organization, _arg, { session, injector }) => {
-    const isAvailable = await injector.get(GitHubIntegrationManager).isAvailable({
-      organizationId: organization.id,
-    });
-    if (!isAvailable) {
+    const isEnabled = injector.get(GitHubIntegrationManager).isEnabled();
+
+    if (!isEnabled) {
```

Comment:
I mixed up `isAvailable` with `isEnabled`... 😄 

`isAvailable` checks if a GitHub integration exists for the organization.

`isEnabled` checks whether the GitHub integration feature is enabled on Hive (self-hosters can run Hive without GitHub).